src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderService.java

```diff
@@ -22,23 +24,31 @@
 @Slf4j
 public class OffenderService {
 
-    private final OffenderRestClient client;
+    private final OffenderRestClient defaultClient;
+    private final AssessmentsRestClient assessmentsClient;
 
     private final Predicate<OffenderDocumentDetail> documentTypeFilter;
 
-    public OffenderService(final OffenderRestClient client, final DocumentTypeFilter documentTypeFilter) {
-        this.client = client;
+    public OffenderService(final OffenderRestClient defaultClient,
+                           final AssessmentsRestClient assessmentsClient,
+                           final DocumentTypeFilter documentTypeFilter) {
+        this.defaultClient = defaultClient;
+        this.assessmentsClient = assessmentsClient;
         this.documentTypeFilter = documentTypeFilter;
     }
 
     public ProbationRecord getProbationRecord(String crn, boolean applyDocumentFilter) {
-
-        Tuple3<ProbationRecord, List<Conviction>, GroupedDocuments> tuple3 = Mono.zip(client.getProbationRecordByCrn(crn), client.getConvictionsByCrn(crn), client.getDocumentsByCrn(crn))
+        Tuple4<ProbationRecord, List<Conviction>, GroupedDocuments, Assessment> tuple4 = Mono.zip(
+            defaultClient.getProbationRecordByCrn(crn),
+            defaultClient.getConvictionsByCrn(crn),
+            defaultClient.getDocumentsByCrn(crn),
+            assessmentsClient.getAssessmentByCrn(crn))
```

Comment:
should we handle errors from the assessments api separately here?

if the CRN isn't found in oasys should we still return the rest of the convictions data without the assessment attached?